lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_server/11_crosscutting_group/amount_calculations.rb

```diff
@@ -0,0 +1,69 @@
+module CarinForBlueButtonTestKit
+  class AmountCalculationTest < Inferno::Test
+    title 'Properly calculates "amount" fields'
+
+    description <<~DESCRIPTION
+      For:
+        -  C4BB ExplanationOfBenefit Inpatient Institutional `.total.amount`
+        -  C4BB ExplanationOfBenefit Inpatient Institutional `.item.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Inpatient Institutional `.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Outpatient Institutional `.total.amount`
+        -  C4BB ExplanationOfBenefit Outpatient Institutional `.item.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Outpatient Institutional `.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Oral `.total.amount`
+        -  C4BB ExplanationOfBenefit Oral `.item.adjudication.amount`
+        -  C4BB ExplanationOfBenefit Professional NonClinician `.total.amount`
+        -  C4BB ExplanationOfBenefit Professional NonClinician `.item.adjudication.amount`
+
+      The Health IT Module calculates the total amount as follows:
+           - Eligible amount = submitted amount - the noncovered amount - discount.
+
+      The subscriber pays the member liability = deductible + coinsurance + copay + noncovered.
+
+      (Part of the member liability may have already been paid to the provider as paidbypatient).
+
+      The eligible amount - the member liability is the payment amount to the provider (paidtoprovider) or the subscriber (paidtopatient).
+    DESCRIPTION
+
+    id :amount_calculations
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@143',
+                          'hl7.fhir.us.carin-bb_2.0.0@154',
+                          'hl7.fhir.us.carin-bb_2.0.0@155',
```

Comment:
143, 154, 155 are all out of scope

Reply:
modified to be in scope. let Karl know. Will regenerate files.